hw3/Sem03Task16.py

```diff
@@ -0,0 +1,15 @@
+# Задача 16:
+# Требуется вычислить, сколько раз встречается некоторое число X в массиве A[1..N].
+# Пользователь в первой строке вводит натуральное число N – количество элементов в массиве.
+# В последующих  строках записаны N целых чисел Ai. Последняя строка содержит число X
+
+# спрашиваем у пользователя исходные данные:
+findMe = input("Введите число для поиска:")
+elements = str.split(input("Введите список элементов через пробел:"))
```

Comment:
Лучше использовать метод, обращаясь к объекту, а не к классу.
```suggestion
elements = input("Введите список элементов через пробел:").split()
```